shims/spark34/src/main/scala/io/glutenproject/sql/shims/spark34/Spark34Shims.scala

```diff
@@ -130,6 +130,33 @@ class Spark34Shims extends SparkShims {
       @transient locations: Array[String] = Array.empty): PartitionedFile =
     PartitionedFile(partitionValues, SparkPath.fromPathString(filePath), start, length, locations)
 
+  override def handleBloomFilterFallback(plan: SparkPlan)(fun: SparkPlan => Unit): Unit = {
```

Comment:
This method looks repeated. If so, maybe, we can just move this method into a common place. But just move the check for `BloomFilterAggregate` into shims, as I note it doesn't exist in spark3.2.

Spark 3.3/3.4:
```
def hasBloomFilterAggregate(agg): Boolean = {
  agg.aggregateExpressions.exists( expr.aggregateFunction.isInstanceOf[BloomFilterAggregate])
}
```
Spark 3.2:
```
def hasBloomFilterAggregate(agg): Boolean = {
   false
}
```

Reply:
Updated, thanks,